pkg/apis/acid.zalan.do/v1/postgresql_type.go

```diff
@@ -225,9 +226,48 @@ type Sidecar struct {
 // UserFlags defines flags (such as superuser, nologin) that could be assigned to individual users
 type UserFlags []string
 
+type Conditions []Condition
+
+type ConditionType string
+type VolatileTime struct {
+	Inner metav1.Time `json:",inline"`
+}
+
+// MarshalJSON implements the json.Marshaler interface.
+func (t VolatileTime) MarshalJSON() ([]byte, error) {
+	return t.Inner.MarshalJSON()
+}
+
+// UnmarshalJSON implements the json.Unmarshaller interface.
+func (t *VolatileTime) UnmarshalJSON(b []byte) error {
+	return t.Inner.UnmarshalJSON(b)
+}
+
+func init() {
+	equality.Semantic.AddFunc(
+		// Always treat VolatileTime fields as equivalent.
+		func(VolatileTime, VolatileTime) bool {
+			return true
+		},
+	)
+}
+
+// Condition contains the conditions of the PostgreSQL cluster
+type Condition struct {
```

Comment:
Can this be changed to PostgresqlCondition? This aligns with other K8s resources.

Reply:
Yeah, changed to `PostgresqlCondition`